### DISCUSSION TITLE
Feature: inner-min, inner-max for QKnob

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12365] [Ideas / Proposals] [answered]
Is it possible to add inner-min, inner-max props to QKnob, like it is done in QSlider:
https://quasar.dev/vue-components/slider
?

![image](https://user-images.githubusercontent.com/70687432/152674410-184ee9a2-4415-4d95-9448-cb558c1e2d3c.png)

**Comment by rstoenescu** [comment, marked as answer]
Implemented inner-min and inner-max for QKnob in Quasar v2.5.4.
